docs/scripts/model_building.rst

```diff
@@ -136,58 +179,97 @@ as a contamination).
   An alternative approach - yet to study - could be to train a classifier with gamma
   against a background sample composed of weighted hadrons and weighted electrons.
 
+The following the example provided by the example configuration file ``RandomForestClassifier.yaml``,
+
 .. code-block:: yaml
 
   General:
-   model_type: 'classifier'
-   # [...] = your analysis local full path OUTSIDE the Vagrant box
-   data_dir: '[...]/shared_folder/analyses/v0.4.0_dev1/data/TRAINING/for_particle_classification/'
-   data_sig_file: 'TRAINING_classification_tail_gamma_merged.h5'
-   data_bkg_file: 'TRAINING_classification_tail_proton_merged.h5'
-   cam_id_list: ['LSTCam', 'NectarCam']
-   table_name_template: '' # leave empty (TO BE REMOVED)
-   outdir: '[...]/shared_folder/analyses/v0.4.0_dev1/estimators/gamma_hadron_classifier'
-
+    # [...] = your analysis local full path OUTSIDE the Vagrant box
+    data_dir: '../../data/' # '[...]/data/TRAINING/for_particle_classification/'
+    data_sig_file: 'TRAINING_classification_tail_gamma_merged.h5'
+    data_bkg_file: 'TRAINING_classification_tail_proton_merged.h5'
+    outdir: './' # [...]/estimators/gamma_hadron_classifier
+    
+    # List of cameras to use (protopipe-MODEL help output for other options)
+    cam_id_list: ['LSTCam', 'NectarCam']
+
+  # If train_fraction is 1, all the TRAINING dataset will be used to train the
+  # model and benchmarking can only be done from the benchmarking notebook
+  # TRAINING/benchmarks_DL2_to_classification.ipynb
   Split:
-   train_fraction: 0.8
-   use_same_number_of_sig_and_bkg_for_training: False  # Lowest statistics will drive the split
-
+    train_fraction: 0.8
+    use_same_number_of_sig_and_bkg_for_training: False  # Lowest statistics will drive the split
+
+  # Optimize the hyper-parameters of the estimator with a grid search
+  # If 'True' parameters should be provided as lists (for None use [null])
+  # If 'False' the model used will be the unique one based on your the
```

Comment:
end of sentence seems to be missing something

Reply:
fixed